package.json

```diff
@@ -4,6 +4,7 @@
   "useWorkspaces": true,
   "workspaces": [
     "packages/compartment-shim",
+    "packages/console",
```

Comment:
I have a conflicting change coming that changes this to `"workspaces": ["packages/*"]`.

Reply:
reconciled.